blogposts/2021/career-framework.md

```diff
@@ -0,0 +1,314 @@
+---
+title: 'How we built our software engineering career framework'
+description: 'Engineering managers and ICs at Sourcegraph once lacked a common language for growth. Here’s the story of how we developed the first version of our first engineering career framework.'
+author: Nick Moore
```

Comment:
@nickmyyz How would you feel about sharing the byline with the contributors? There are some instances of referring to "us" and "we" that are specific to the working group (for example: "We often spent time talking about the specific, minute wordings of different sentences and bullet points") as opposed to speaking as Sourcegraph the company. It feels more authentic to co-credit the contributors in that case? Alternatively, we could attribute the post to them and include a credit to you at the bottom "As told to Nick Moore" or something? Let me know what you think! Thank you also for your patience as we figure this voice business out. I think it will be smoother in future if we make these choices from the outset 😀

Reply:
@rebeccadee I'm open to either option! I lean toward the former for this post but I'll check with the interviewees to see that they think